Cargo.toml

```diff
@@ -40,8 +40,8 @@ prettyplease = "0.2.22"
 trybuild = "1.0"
 
 # members
-mini-alloc = { path = "mini-alloc", version = "0.8.1" }
+mini-alloc = { path = "mini-alloc", version = "0.9.0-rc.1" }
 stylus-sdk = { path = "stylus-sdk" }
-stylus-core = { path = "stylus-core", version = "0.8.1" }
-stylus-test = { path = "stylus-test", version = "0.8.1" }
-stylus-proc = { path = "stylus-proc", version = "0.8.1" }
+stylus-core = { path = "stylus-core", version = "0.9.0-rc.1" }
+stylus-test = { path = "stylus-test", version = "0.9.0-rc.1" }
+stylus-proc = { path = "stylus-proc", version = "0.9.0-rc.1" }
```

Comment:
Any reason why we need to specify all these versions here? Could we just remove them (like stylus-sdk)?